build.sbt

```diff
@@ -97,3 +97,16 @@ lazy val lambdaApiGatewayProxyHttp4s = crossProject(JSPlatform, JVMPlatform)
     )
   )
   .dependsOn(lambda, lambdaEvents)
+
+lazy val cloudformationCustomResource = crossProject(JSPlatform, JVMPlatform)
+  .crossType(CrossType.Pure)
+  .in(file("cloudformation-custom-resource"))
+  .settings(
+    name := "feral-cloudformation-custom-resource",
```

Comment:
Obviously still figuring this stuff out, but so far I'm namespacing AWS stuff as `feral-lambda`. (This is a lambda, right? Or should the namespace we use be aws?)